torch/_library/custom_ops.py

```diff
@@ -52,6 +53,11 @@ def custom_op(
             is valid for. If no device type is provided, then the function
             is used as the default implementation for all device types.
             Examples: "cpu", "cuda".
+        manual_schema (None | str): A schema string for the operator. If not
+            provided, we'll infer a schema for the operator from its type
+            annotations. We recommend letting us infer a schema unless you
+            have a specific reason not to (e.g. you're code-generating a schema).
+            Example: "(Tensor x, int y) -> (Tensor, Tensor)".
```

Comment:
Do we have a description of our schema rule somewhere in aten we can link to? Even if it is a link to some .md or comment on github, I think this is useful.

Reply:
Yes, we've got one